Drop after x11-wm/plasma6-kdecoration >= 6.1.0

https://github.com/winft/como/commit/3155adace1ce
https://invent.kde.org/plasma/kdecoration/-/commit/4612e0f003a3

como/win/deco/settings.h:113:59: error: no member named 'Spacer' in 'KDecoration2::DecorationButtonType'
        s_buttonNames[KDecoration2::DecorationButtonType::Spacer] = QChar('_');
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^

--- como/win/deco/settings.h.orig	2024-06-19 18:07:57 UTC
+++ como/win/deco/settings.h
@@ -110,7 +110,6 @@ class settings : public QObject, public KDecoration2::
         s_buttonNames[KDecoration2::DecorationButtonType::KeepAbove] = QChar('F');
         s_buttonNames[KDecoration2::DecorationButtonType::KeepBelow] = QChar('B');
         s_buttonNames[KDecoration2::DecorationButtonType::Shade] = QChar('L');
-        s_buttonNames[KDecoration2::DecorationButtonType::Spacer] = QChar('_');
     }
 
     QString buttonsToString(QVector<KDecoration2::DecorationButtonType> const& buttons) const
